test/conformance/conformance_concurrent_queue.cpp

```diff
@@ -1207,19 +1207,29 @@ void TestMoveSupportInPushPop() {
 template<class T>
 class allocator: public oneapi::tbb::cache_aligned_allocator<T> {
 public:
+    state_type state;
     std::size_t m_unique_id;
 
-    allocator() : m_unique_id( 0 ) {}
+    allocator() : state(LIVE), m_unique_id(0) {}
+    allocator(size_t unique_id) : state(LIVE) { m_unique_id = unique_id; }
```

Comment:
Given that previous suggestion accepted : 
```suggestion
    allocator() :  m_unique_id(0) {}
    allocator(size_t unique_id) : m_unique_id(unique_id) {}
```